linguatec_lexicon/templates/linguatec_lexicon/exportdata.html

```diff
@@ -0,0 +1,52 @@
+{% extends 'rest_framework/base.html' %}
+{% load i18n static %}
+
+{% block title %}{{ title }}{% endblock %}
+
+{% block branding %}
+<a class="navbar-brand" rel="nofollow" href="/">{{ title }}</a>
+{% endblock %}
+
+{% block content %}
+
+<div style="width: 45%; float:left">
```

Comment:
Instead of withd and float, I think that it's better to use **bootstrap layout** (rows + cols). I'm not sure which version is included with django-rest-framework.

Reply:
Trying to use a contatiner with one row and two columns but for whatever reason it is not working, but maybe could be better to be displayed in rows